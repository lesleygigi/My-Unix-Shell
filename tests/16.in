/bin/uname > /tmp/a.b &
wait
seq 3
