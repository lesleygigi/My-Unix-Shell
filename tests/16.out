/bin/uname > /tmp/a.b &
wait
seq 3
1
2
3
